MobileFilter/sections/specific_app.txt

```diff
@@ -18,6 +18,7 @@
 ||graph.facebook.com^$app=com.booking
 !
 ! Blocking Facebook mobile ads
+||graph.facebook.com^$app=com.word.game.crossword.brain.puzzle.scapes.connect
```

Comment:
Sadly I cannot install. Is the app allowing the user to login via facebook? If yes, the rule will break it.

Reply:
it is ok